docs/release-notes/spring-2022/README.md

```diff
@@ -11,53 +11,59 @@
 
-
-## New Features
-
-### New X Icons
-
 ## Bug Fixes
 
+- CdrPopover has been updated to address an issue where the component would emit its `open` event twice in certain circumstances.
 ## Deprecations
 
-### Vue2
-In preparation of our Vue3 update the existing vue2 cedar library is now deprecated. As this is a major update that may impact teams differently we will continue our support of these assets for an additional release.
-Additionally, we plan to archive the vue2 repository as "REI-Cedar-Vue2" and publish make this available vis NPM.
+### Vue 2
 
-## Breaking Changes
+In preparation of our Vue 3 migration we are deprecating the Vue 2 based component library. We will support this final release of the Vue 2 library with basic maintainence and bug fixes for an additional 12 months. Once we release our Vue 3 library, we will re-publish our Vue 2 libray as `@rei/cedar-vue-2`.
 
+## Breaking Changes
 ### Removals
 
-#### CdrBreadcrumb
-Scoped Slots have been removed
+#### Scoped Slots
+
+Scoped slots have been removed from CdrBreadcrumb and CdrPagination. See the [CdrBreadcrumb](../../components/breadcrumb/#custom-navigation) or [CdrPagination](../../components/pagination/#overriding-default-navigation) pages for examples of how to override their default navigation behavior.
+
+#### CdrBanner Component Variable
+
+The CdrBanner component variable has been removed. Teams who are unable to utilize the Vue component should plan to make use of the [messaging tokens](../../tokens/all-tokens/#colors) to create their banners.  
+
+### Preparing for Vue 3
 
-#### CdrPagination
-Scoped Slots have been removed
+*The following have been carried over from the [Spring 2021 release](../spring-2021/#deprecations). We will continue to note Vue 3 related deprecations so teams can adequately prepare for the upcoming migration.*
+#### Vue 3: Update Slot Syntax
 
+Vue 2.6 introduced a new `v-slot` syntax for passing slot content into components. A pound sign `#` can be used as a shorthand for `v-slot:`, much like a colon `:` can be used as a shorthand for `v-bind`. The old syntax is removed from Vue 3 and we recommend updating your codebase to make use of the new slot syntax to simplify the upgrade process in the future. Note that the new `v-slot` or `#` syntax can only be used on a `template` tag, however those additional `template` tags will not be included in the rendered HTML.
 
-## Preparing for Vue 3
 
-### Release Candidate plan
+## Documentation Refresh
 
-### Composition API
+- We have updated our "Getting Started" section to "Guides" to better reflect that this content is for ongoing support, not just onboarding to the system.
 
-### Resources 
+### For Designer
 
+- We have released a new "for designers" resource page. 
+### Developer
 
+- We have released a new "for developers" resource page.
```

Comment:
– Our new resource page for developers includes Vue guides, troubleshooting tips, and definitions of Cedar's parts.